### ISSUE TITLE
[Item]Tiny Abomination in a Jar motes/attacks disappearing

### ISSUE THREAD

**Issue by Chenocide** [dalaranwow/dalaran-wow#4972]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

The number of generated motes during a fight, does not match with the expected number of attacks made by this trinket. 

**Expected behaviour**: Tell us what should happen instead

All or nearly all attacks should hit a target and motes shouldn't simply disappear during a fight.

**Steps to reproduce the problem**:

1. Equipt Tiny Abomination in a Jar.
2. Participate in a boss fight.
3. Measure applied buffs and damage done.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

I'll admit that I have no concrete proof of WHAT the problem is, I have only noticed the symptoms. 
In 3.3.0, the trinket in question had issues with range. The range on the generated attack would only actually hit something if you were extremely close to the target:

"As others have said, TAJ requires you to be standing right on top of the target of the proc (less than 2 yds)."

http://www.mmo-champion.com/threads/688878-Tiny-Abomination-in-a-Jar-test-results-w-pic

This is currently my only real hint at the underlying problem. 
According to logs, about 66% of the motes/attacks disappears.

https://puu.sh/y8sVK/31dfd89060.png
https://puu.sh/y8sUW/94c074a085.png

![alt text](https://puu.sh/y8sVK/31dfd89060.png)
![alt text](https://puu.sh/y8sUW/94c074a085.png)

On Blood Queen, 9 instances of Tiny reaching 8 stacks and dealing damage but apparently I gained a mote 255 times, according to the "Buffs gained" tab. 9 instances of damage would require 72 motes of anger. I realize that Logs are inaccurate but it really seems like a red flag to me. The above numbers were done by a enhancement shaman, similar but not as severe numbers can be seen by other classes, 

If range is really the issue, then it will obviously depend a lot on the player and positioning. Maybe @TeridanDW can test this out and confirm that either something weird is going on or it's just me being paronoid? 

Logs where I got the numbers from, I only had the trinket on Blood Princes and BQL: 
http://www.worldoflogs.com/reports/rt-qaszoo0nsum5rki3/

IDs:
25normal: 50351
25heroic: 50706

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Chenocide.
I will take a look, just 1 question, is the log from 25N or 25HC version of the trinket?

**Comment by Chenocide** [comment]
@TeridanDW But it's not like a normal miss, it doesn't even register any misses. Honestly, assuming that it's a miss is just the best explanation, cause why would 2/3 of the attacks just disappear? I can do more raids to get more data and I can try to go through more recorded logs and find more proof of this thing not working as it should.

**Comment by Chenocide** [comment]
@TeridanDW It seems I've made a huge mistake, I didn't take critical strikes into account. Now every example I look at, I end up with more motes required than what is actually listed. I don't know how this trinket works. Feel free to close the report, I don't consider it accurate anymore.

**Comment by TeridanDW** [comment]
Alright, if you find any proof of an issue post it below and we will look at it again, thank you.
